src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrDate = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
The assumption that the delimiter is the last element of the `fromFormat` array may not always be correct. Consider handling cases where the delimiter is different or not explicitly specified.